Ecommerce/ProductService/Validators/ProductValidator.cs

```diff
@@ -0,0 +1,19 @@
+﻿using FluentValidation;
+using ProductService.Models;
+
+namespace ProductService.Validators
+{
+    public class ProductValidator : AbstractValidator<Product>
+    {
+        private readonly int maxNameLength = 100;
+        private readonly int minPrice = 0;
+        private readonly int minStock = 0;
+
+        public ProductValidator() 
+        {
+            RuleFor(product => product.Name).NotEmpty().MaximumLength(maxNameLength);
+            RuleFor(product => product.Price).GreaterThanOrEqualTo(minPrice);
```

Comment:
Я бы поменял GreaterThanOrEqualTo на GreaterThan, чтобы нельзя было создать продукт, который стоит 0
```suggestion
            RuleFor(product => product.Price).GreaterThan(minPrice);
```